### ISSUE TITLE
postgresql schema

### ISSUE THREAD

**Issue by blackpawninja** [samonzeweb/godb#42]
i use schema to group tables in my db.  i usually write my query like this
```
SELECT * FROM auth.user
```
so i write a TableName() like this
```
func (*User) TableName() string { return "auth.user" }
```

but postgresql complains that relation "auth.user" does not exists. initial digging suggest that the `Quote()` function in godb simply apply quote around identifier, resulting in `"auth.schema"`. while proper postgresql schema identifier should be `"auth"."schema"`

if i want to contribute a fix, should i modify the adapter's `Quote()` function or somewhere else?

**Comment by samonzeweb** [comment]
Hi,

You're right, I've started with a very naive quoting approch and just hacked a bit around it later. 

Thanks for your proposition. But I have to consider all cases before replying to you. Then I made the change myself as it's not a big work once the code was checked.

You were almost right about the Quote method, but I left it as-is, and share the split & merge work as it's common to all adapters. In fact the code already exists but just for some cases.

As help is always appreciate, and I don't want to break something, you can :
* do a review of my pull request : https://github.com/samonzeweb/godb/pull/43
* try the code using `go get github.com/samonzeweb/godb@betterquote` (only with go modules, not with GOPATH)

I will not be available tomorrow, but I'll try to react ASAP.

Sam.

**Comment by blackpawninja** [comment]
the PR looks good to me. i also tested `betterquote` branch and it works with my existing code

thank you

**Comment by samonzeweb** [comment]
PR merged, see version tag v1.0.13 .
